src/controllers/profile.js

```diff
@@ -0,0 +1,110 @@
+import { config } from 'dotenv';
+import cloudinary from 'cloudinary';
+import { User } from '../sequelize/models';
+
+config();
+cloudinary.config({
+  cloud_name: process.env.cloudinaryName,
+  api_key: process.env.cloudinaryApiKey,
+  api_secret: process.env.cloudinarySecretKey,
+});
+
+/**
+ * @description User profile
+ */
+class Profile {
+  /**
+       * @description get user profile
+       * @param {object} req
+       * @param {object} res
+       * @returns {object} return object containing user profile
+       */
+  static async user(req, res) {
+    const { userName } = req.params;
+    try {
+      const userfound = await User.findOne({ where: { userName } });
+      if (!userfound) {
+        return res.status(404).json({
+          message: `profile for ${userName} not found`,
+        });
+      }
+      res.status(200).json({
+        message: 'successfully retrieved a user profile',
+        data: {
+          userName: userfound.userName,
+          bio: userfound.bio,
+          image: userfound.image,
+        }
+      });
+    } catch (error) {
+      // console.log(error);
+    }
+  }
+
+  /**
+       * @description update user profile
+       * @param {object} req
+       * @param {object} res
+       * @returns {object} return object containing updated user profile
+       */
+  static async editProfile(req, res) {
```

Comment:
If I see this function, It is long. I think it would be better if cloudinary function is in middleware.